src/aws-sdk/sdk.ts

```diff
@@ -84,6 +84,7 @@ const SFN_SERVICE_BLOCK_LIST: {
     "skipWaitTimeForInstanceTermination",
   ],
   ComprehendMedical: ["detectEntities"],
+  ControlTower: true,
```

Comment:
It's not explicitly documented as supported or unsupported. Maybe we should test it? This would be amazing for building an org-formation like experience